pdf-inspector.gemspec

```diff
@@ -16,6 +16,8 @@ Gem::Specification.new do |spec|
                 "jimmy@deefa.com"]
   spec.licenses = %w(PRAWN GPL-2.0 GPL-3.0)
   spec.add_dependency('pdf-reader', '~>1.0')
+  spec.add_development_dependency('rspec')
+  spec.add_development_dependency('simplecov')
```

Comment:
I suppose if I'm not building up a Gemfile nor a gem itself, there's no reason to have them there.